### DISCUSSION TITLE
Adding a synchronisation method between devices (phone, tablet, etc...)

### DISCUSSION THREAD

**Discussion by EdoardoCortivo** [aniyomiorg/aniyomi#1548] [Ideas]
On Lemuroid (an Android app written in kotlin for playing retro games) there is a synchronization option via Google drive so that you can start playing on the tablet and finish on your phone, why not adding the same feature so that you can start an Anime on the tablet and finish it on the phone? This can be implemented by using a button that saves a backup on the drive linked to the app and when you open the app it automatically search if there is a new backup and in that case restore that backup.

**Comment by sBaydin** [comment]
#1510 same here

**Comment by REFLEXGAMING007** [comment]
Add SyncYomi support to Aniyomi please 🙏
